corehq/apps/app_manager/suite_xml/case_tile_templates/icon_text_grid.json

```diff
@@ -0,0 +1,22 @@
+{
+    "slug": "icon_text_grid",
+    "filename": "icon_text_grid.xml",
+    "has_map": false,
+    "fields": ["top_left_image", "top_left_text", "top_right_image", "top_right_text",
+      "middle_left_image", "middle_left_text", "middle_right_image", "middle_right_text",
+      "bottom_left_image", "bottom_left_text"],
+    "grid": {
+        "top_left_image": {"x": 0, "y": 0, "width": 1, "height": 1, "horz-align": "left", "vert-align": "center"},
```

Comment:
nit: should this horz-align also be 'center'?

Reply:
you are right. I was trying something out.